### ISSUE TITLE
Fibrovascular papilloma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10071]
**Preferred term label:**
Fibrovascular papilloma

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0334110

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Intraductal papilloma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this a well delineated type of breast neoplasm? As far as I can see it is not included in NCIT and I cannot find a good definition.

**Comment by MickeySegal** [comment]
I see that our term got changed to "Breast biopsy: fibrovascular cores".  

Does that work?

**Comment by pnrobinson** [comment]
We are in the process of adding a new section to HPO with biopsy findings for cancer and other diseases. I think this will need to go there, but it may take some more time.

**Comment by pnrobinson** [comment]
@MickeySegal 
I think that probably we would need to specify what kind of fibrovascular core(s) we are talking about. Maybe this article is a good start. Let me know if this is still relevant and I will add a mini hierarchy
https://aplm.kglmeridian.com/view/journals/arpa/140/7/article-p628.xml

**Comment by MickeySegal** [comment]
This looks like what we mean.  We have this finding listed now for 2 diseases:

- Intraductal papilloma
- Breast ductal carcinoma in situ
